Tasks/MavenV2/maventask.ts

```diff
@@ -337,7 +338,8 @@ function publishJUnitTestResults(testResultsFiles: string) {
     }
 
     var tp = new tl.TestPublisher("JUnit");
-    tp.publish(matchingJUnitResultFiles, true, "", "", "", true);
+    const testRunTitle = tl.getInput('testRunTitle');
```

Comment:
I think this can be `null`.  Might want to add `|| ""`.

Reply:
The `publish` method already does null checks on the inputs